src/ProjectOrigin.WalletSystem.Server/Models/WalletAttribute.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace ProjectOrigin.WalletSystem.Server.Models;
+
+public record WalletAttribute
+{
+    public required Guid CertificateId { get; init; }
+    public required string RegistryName { get; init; }
+    public required string Key { get; init; }
+    public required string Value { get; init; }
+    public required byte[] Salt { get; init; }
+
+    public string GetHashedValue()
+    {
+        var str = Key + Value + CertificateId.ToString() + Convert.ToHexString(Salt);
```

Comment:
Why add key and certificate id?

Reply:
It was a recommendation from the more cryptographic knowledgable people https://github.com/project-origin/registry/issues/129

Reply:
Okay, that's fair. But I am thinking about the clients that is going to use this. Will they have to create the hashed value in the same way?

Reply:
Only issuers, everyone else will be ignorant of the fact, all of this begins to speak for an IssuerService that could help remove some of this from an issuers code? like the wallet those for the other aspects.

Reply:
Yes, issuing is not simple - or it is quite easy to make mistakes. I also see that there is a need for an IssuerService. Would be nice to also have Elia's input on this.